src/components/Composer/index.js

```diff
@@ -283,6 +287,21 @@ class Composer extends React.Component {
         event.stopPropagation();
     }
 
+    putSelectionInClipboard(event) {
+        // If anything happens that isn't cmd+c or cmd+x, ignore the event because it's not a copy command
+        if (!event.metaKey || (event.key !== 'c' && event.key !== 'x')) {
+            return;
+        }
```

Comment:
@tgolen Here we are only checking for the `metaKey` modifier which is only available on macOS. I'm not sure if this is intended but I think we should also check for `ctrlKey` to have same behaviour on other OS (Windows & Linux).

Reply:
Sounds fine to me, but it's very low value since we don't officially support those platforms. Just my thoughts

Reply:
Actually we are removing the whole `putSelectionInClipboard` function here https://github.com/Expensify/App/pull/19738. We no longer need it as the issue that this function was meant to solve is already fixed by https://github.com/Expensify/App/pull/15034 and the original issue https://github.com/Expensify/App/issues/12769 is no longer reproducible.